[2024-04-18T08:24:11.856+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-18T08:24:11.876+0000] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: trigger_airbyte_job_example.airbyte_trigger_google_sheets_sync scheduled__2024-04-16T00:00:00+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2024-04-18T08:24:11.893+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
